packages/@okta/vuepress-site/docs/reference/system-log-query/index.md

```diff
@@ -0,0 +1,510 @@
+| policy.rule.activate          | A rule in a policy is activated   |
+| policy.rule.add               | A rule is added to a policy       |
+| policy.rule.deactivate        | A rule in a policy is deactivated |
+| policy.rule.delete            | A rule is deleted from a policy   |
+| policy.rule.update            | A rule in a policy is updated     |
+
+#### Policy event details
+
+* `policy.evaluate_sign_on` provides context on the values that are used and evaluated in the context of the Okta sign-on policy. For example, you can determine which network zones are matched for this event.
+* For `policy.lifecycle` and `policy.rule` events, the corresponding policy is listed in the target object.
+
+### Rate limit events
+
+See [System Log events for rate limits](/docs/reference/rl-system-log-events/) for information on rate limit Event Types.
+
+Rate limit warnings are sent at different times, depending on the org type. For One App and Enterprise orgs, the warning is sent when the org is at 60% of its limit.
+
+> **Note:** For orgs created before 2018-05-17, the warning is sent at 90%.
+
+Rate limit violations are sent when a rate limit is exceeded.
+
+### Security events
+
+| Event                    | Description                                                                                   |
+| :-------------------     | :----------------------------------                                                           |
+| security.request.blocked | A request is blocked due to a blocklist rule (such as an IP network zone or location rule). |
+
+### User events
+
+| Event                       | Description                                               |
+| :-------------------------- | :-------------------------------------------------------- |
+| user.authentication.sso     | A user attempts a Single Sign-On (SSO) to an app managed in Okta |
+| user.lifecycle.activate     | A user account is activated                             |
+| user.lifecycle.create       | A user account is created                               |
+| user.lifecycle.deactivate   | A user account is deactivated                           |
+| user.lifecycle.suspend      | A user account is suspended                             |
+| user.lifecycle.unsuspend    | A user account is moved from suspended status           |
+| user.session.start          | Okta issues a session to a user who is authenticating    |
+
+#### User event details
+
+`user.authentication.sso` doesn't capture whether the SSO attempt is successful or has failed, because Okta can't collect the subsequent authentication attempt status from the third-party service.
+
+## Event correlation
+
+When reviewing the System Log, it's often useful to correlate events to understand the thread of events that have occurred at a particular time.
+
+The response object offers two identifiers in this respect:
+  - `authenticationContext.externalSessionId`: Identifies events that occurred in the same user session
```

Comment:
Let's change this from two to three identifiers and add 

`authenticationContext.rootSessionId`: Identifies events from all sessions that share a common root session. Some actions that can be attributed to a user, such as those taken by an Okta system actor on behalf of a user may have a different `externalSessionId` value. The `rootSessionId` field allows all of those to be identified and grouped together.